CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/analytics/GraphAnalyticPlugin.java

```diff
@@ -60,6 +62,13 @@ protected final void computeResultsFromGraph(final GraphReadMethods graph, final
         });
     }
 
+    public String getHelpPath() {
+        final String CODEBASE_NAME = "constellation";
+        final String sep = File.separator;
+        return Generator.getBaseDirectory() + CODEBASE_NAME + sep + "CoreAnalyticView" + sep + "src" + sep + "au" + sep + "gov" + sep
+                + "asd" + sep + "tac" + sep + CODEBASE_NAME + sep + "views" + sep + "analyticview" + sep + "docs" + sep;
+    }
```

Comment:
Wondering if this function could be lifted up another level (i.e. `AnalyticPlugin`) since it is the same implementation in the other analytic plugin types as well (e.g. `ScoreAnalyticPlugin`). I would also rename `CODEBASE_NAME` to `codebaseName` if it is going to be a local variable rather than static constant